playbooks/10 - SP - FortiSOAR Health Assessment/Referenced > Generate Configuration.json

```diff
@@ -0,0 +1,213 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Referenced > Generate Configuration",
```

Comment:
As discussed, update the playbook names as per standard nomenclature

Reply:
Updated